rest/rest-client/build.gradle

```diff
@@ -1,7 +1,8 @@
 apply plugin: 'java-library-distribution'
 
 dependencies {
-    compile 'org.ow2.proactive:catalog-client:+'
+    compile "org.ow2.proactive:catalog-client:${schedulingVersion}"
+    compile group: 'org.springframework', name: 'spring-web', version: '4.2.5.RELEASE'
```

Comment:
why don t we just add the dependency in the catalog-client?

Reply:
The dependency is in the catalog-client. It seems that because the generated methods throw a RestClientException from the spring-web dependencies, the dependency must be added. If the dependency is missing or the keyword "compile" is replaced by "runtime", it doesn't compile.